src/test/signalpathtest.h

```diff
@@ -142,6 +145,18 @@ class BaseSignalPathTest : public MixxxTest, SoundSourceProviderRegistration {
         PlayerInfo::destroy();
     }
 
+#ifdef __RUBBERBAND__
+    void SetUp() override {
+        RubberBandWorkerPool::createInstance();
+    }
+#endif
+
+#ifdef __RUBBERBAND__
+    void TearDown() override {
+        RubberBandWorkerPool::destroy();
+    }
+#endif
```

Comment:
I don't understand what you want me to do here. Could you elaborate?

Reply:
otherwise rubberband-less builds fail because otherwise the code further below will try to call the nonexistent methods.
```suggestion
    void SetUp() override {
#ifdef __RUBBERBAND__
        RubberBandWorkerPool::createInstance();
#endif
    }

    void TearDown() override {
#ifdef __RUBBERBAND__
        RubberBandWorkerPool::destroy();
#endif
    }
```